app/config/config.php

```diff
@@ -146,6 +146,20 @@
     'schema_filter'  => '~^(?!'.MAUTIC_TABLE_PREFIX.'messenger_messages)~',
 ];
 
+if (!empty($localConfigParameterBag->get('db_host_ro'))) {
+    $dbalSettings['wrapper_class']   = \Mautic\CoreBundle\Doctrine\Connection\PrimaryReadReplicaConnectionWrapper::class;
+    $dbalSettings['replicas']        = [
```

Comment:
Did you intentionally remove the `keep_slave`/`keep_replica` settings? I'm not sure if this is necessary, just curious.

Reply:
I think we need it, I removed it while I was testing, so it went missing. I added it back